src/msw/printdlg.cpp

```diff
@@ -911,12 +927,19 @@ bool wxWindowsPrintDialog::ConvertToNative( wxPrintDialogData &data )
     if ( data.GetEnableHelp() )
         pd->Flags |= PD_SHOWHELP;
 
+#if(WINVER >= 0x0500)
```

Comment:
We never use `WINVER` checks in wx code. This particular one is obsolete and can be simply removed anyhow because we only support Windows 7 or later (and probably won't support even Windows 7 for much longer) in master, but if you ever need to perform any checks, they need to be done during run-time and not compile-time because we want the same wx DLL to work on all Windows versions.

Reply:
Ok, changed.